### PR TITLE
typescript `src/video-grid`

### PR DESCRIPTION
**Pull request by toger5**

### CODE REVIEW

**Review by SimonBrandner — COMMENTED** [review]
Thank you!

**Review by SimonBrandner — APPROVED** [review]

File: src/video-grid/VideoTile.tsx [hunk 1]
```diff
@@ -17,28 +17,52 @@ limitations under the License.
...
+
 import styles from "./VideoTile.module.css";
 import { ReactComponent as MicMutedIcon } from "../icons/MicMuted.svg";
 import { ReactComponent as VideoMutedIcon } from "../icons/VideoMuted.svg";
 import { AudioButton, FullscreenButton } from "../button/Button";
 
-export const VideoTile = forwardRef(
+interface Props {
+  name: string;
+  showName?: boolean;
```
Comment (SimonBrandner) [inline src/video-grid/VideoTile.tsx:28]: I think we actually don't use this anywhere now

Comment (toger5) [reply src/video-grid/VideoTile.tsx:28]: Let me double check where it is used. But yea that could be right. Sorry for merging then. It felt like the merge didnt change anything needing a review 😬 my bad

Comment (toger5) [reply src/video-grid/VideoTile.tsx:28]: Its still used in `VideoGrid.stories.tsx`:
```tsx
<VideoGrid layout={layout} items={items}>
          {({ item, ...rest }) => (
            <VideoTile
              key={item.id}
              name={`User ${item.id}`}
              showName={items.length > 2 || item.focused}
              disableSpeakingIndicator={items.length < 3}
              {...rest}
            />
          )}
        </VideoGrid>
```
But you are right it has no effect? Should we remove it properly from everywhere?

Comment (SimonBrandner) [reply src/video-grid/VideoTile.tsx:28]: Yes, we should - I did it this morning in other places :p See https://github.com/vector-im/element-call/pull/514

**Review by SimonBrandner — COMMENTED** [review]
Clearing review

File: src/video-grid/VideoTile.tsx (hunk 1 above)
Comment (toger5) [reply src/video-grid/VideoTile.tsx]: Okay updated the PR

**Review by SimonBrandner — APPROVED** [review]

**Pull request merged**